src/ffi/cred_offer.rs

```diff
@@ -29,5 +29,26 @@ pub extern "C" fn anoncreds_create_credential_offer(
     })
 }
 
+/// Create Credential Offer according to the AnonCreds specification
```

Comment:
Do we not want to scope this under `src/ffi/w3c/cred_offer`? Makes the feature flag a bit easier as well.